src/librustdoc/html/render/search_index.rs

```diff
@@ -243,12 +244,14 @@ fn add_generics_and_bounds_as_types<'tcx>(
     tcx: TyCtxt<'tcx>,
     recurse: usize,
     res: &mut Vec<TypeWithKind>,
+    cache: &Cache,
```

Comment:
Can you add `cache` to the `skip` list in the above `instrument` attribute? That'll allow you to remove the derive of `Debug`.

Reply:
solved, please check.

Reply:
Looks good!